### PR TITLE
Add Verify Bulk Job to CLI (JAVACLI-74)

### PR DESCRIPTION
**Pull request by berkeleysquare**

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        if (!Guard.isStringNullOrEmpty(args.getObjectName())) {
+            System.out.println("Warning: '-o' is not used with verify and is ignored.");
+        }
+        this.prefix = args.getPrefix();
+
+        return this;
+    }
+
+    @Override
+    public VeirfyBulkJobResult call() throws Exception {
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java]: speeling error in veirfy

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import com.spectralogic.ds3client.models.Contents;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.utils.Guard;
+import org.apache.commons.cli.MissingOptionException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VerifyBulkJob extends CliCommand<VeirfyBulkJobResult> {
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java]: *VerifyBulkJobResult

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            final List<Ds3Object> objectList = new ArrayList<>();
+            for (final Contents contents : bulkContents) {
+                objectList.add(new Ds3Object(contents.getKey(), contents.getSize()));
+            }
+
+            // Make verify call to API
+            final VerifyBulkJobSpectraS3Response verifyResponse
+                    = getClient().verifyBulkJobSpectraS3(new VerifyBulkJobSpectraS3Request(this.bucketName, objectList ));
+
+            return new VeirfyBulkJobResult(this.bucketName, verifyResponse.getMasterObjectListResult().getObjects().iterator());
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java]: hear to

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.utils.Guard;
+import org.apache.commons.cli.MissingOptionException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VerifyBulkJob extends CliCommand<VeirfyBulkJobResult> {
+
+    private String bucketName;
```
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java]: missing final

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    private String prefix;
```
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java]: missing final

Comment (berkeleysquare) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/VerifyBulkJob.java]: ?? But they can be reassigned -- in fact they have to.

**Pull request merged**